python/adaptivity.py.in

```diff
@@ -414,13 +472,12 @@ def adapt(metric, bfaces=None, bfaces_func=None, colinearity_tol = 30, debugon=T
       info("Colinearity detection ...")  
       bfaces_func = detect_colinearity(mesh, tol = colinearity_tol)
 
-  bfaces_IDs = numpy.array([bfaces_func[f.index()] for f in bfaces_list], dtype = numpy.int32)
+  bfaces_IDs = numpy.array([bfaces_func[f.index()] for f in bfaces_list], dtype = numpy.intc)
     
   x = coords[nodes,0]
   y = coords[nodes,1]
   if element.cell().geometric_dimension() == 3:
     z = coords[nodes,2]
-  cells = array(cells,dtype=numpy.intc)
```

Comment:
I will, sorry, many people worked on this interface before me and I missed this. cells has to be avoided as a variable name because it is a function in dolfin that would then get overwritten.